### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by mkholjuraev**
The PR is entended to revert previous commit and to resolve test failures with more clean code. The previous commit had a lot of dirty code due to last minute urgent deployment.

### CODE REVIEW

File: src/SmartComponents/Systems/System.test.js [hunk 1]
```diff
@@ -1,171 +1,168 @@
...
+    };
+
+    const mockStore = configureStore([customMiddleWare]);
+    return mockStore(state);
+};
+
+let wrapper;
+let store = initStore(mockState);
+
+beforeEach(() => {
```
Comment (bastilian) [inline src/SmartComponents/Systems/System.test.js:89]: These could go `beforeEach` & `afterEach` can go within the describe block, or?

File: src/SmartComponents/Systems/System.test.js [hunk 2]
```diff
@@ -1,171 +1,168 @@
...
+    console.error = () => { };
```
Comment (bastilian) [inline src/SmartComponents/Systems/System.test.js:90]: hmm.... ? :D

File: src/SmartComponents/Packages/Packages.test.js [hunk 3]
```diff
@@ -48,47 +48,47 @@ const initStore = (state) => {
 let wrapper;
 let store = initStore(mockState);
 
-// beforeEach(() => {
-//     console.error = () => { };
+beforeEach(() => {
+    console.error = () => { };
```
Comment (bastilian) [inline src/SmartComponents/Packages/Packages.test.js:52]: Screenshot 2022-03-09 at 12 20 47

:D

Comment (mkholjuraev) [reply src/SmartComponents/Packages/Packages.test.js:52]: :)

File: src/Utilities/api.js [hunk 4]
```diff
@@ -23,7 +23,7 @@ export function createApiCall(
             data,
             ...requestConfig
         })
-    );
+    ).catch((err) => err);
```
[the remark this answers is not in the document]
Comment (mkholjuraev) [reply src/Utilities/api.js:26]: hmm, that makes sense. I will change to notification.

File: src/SmartComponents/Systems/System.test.js (hunk 2 above)
Comment (mkholjuraev) [reply src/SmartComponents/Systems/System.test.js:90]: I remember terminal was full of error logs which are repeated and I had to scroll a lot. Thus I disabled errors. I find it easy to debug the test results with all the failing test messages. Yes it is time to bring error handling to standards. I will delete all the occurrances.

File: src/SmartComponents/Advisories/Advisories.test.js [hunk 5]
```diff
@@ -130,58 +128,52 @@ describe('Advisories.js', () => {
...
+            selected: 'RHEA-2020:2743'
         });
     });
 
     it('should handle remediation', () => {
         const remediationProvider = wrapper.find('TableView').props().remediationProvider;
-        remediationProvider().catch(() => {});
+        remediationProvider().catch((err) => console.log(err));
         expect(fetchSystems).toHaveBeenCalledWith({ limit: -1 });
-        //expect(fetchViewAdvisoriesSystems).toHaveBeenCalledWith('');
```
[the remark this answers is not in the document]
Comment (mkholjuraev) [reply src/SmartComponents/Advisories/Advisories.test.js:184]: In the final commit this line is deleted.

File: src/SmartComponents/Systems/System.test.js (hunk 1 above)
Comment (mkholjuraev) [reply src/SmartComponents/Systems/System.test.js:89]: Yup

File: src/SmartComponents/Systems/System.test.js (hunk 2 above)
Comment (bastilian) [reply src/SmartComponents/Systems/System.test.js:90]: I think silencing `console.error` is fine to some degree. Here I'd just be afraid of someone trying to debug something, not knowing that it is silenced in the `beforeEach` at the top.

**Review by bastilian — APPROVED** [review]
LGTM! Thank you @mkholjuraev!

**Comment by mkholjuraev** [comment]
@bastilian you are always helpfull, thanks!

**Pull request merged**